packages/sdk/src/browser/index.ts

```diff
@@ -157,10 +247,11 @@ class BrowserClient {
     window.addEventListener("message", onMessage);
 
     try {
-      this.createIframe(opts);
+      await this.createIframe(opts);
     } catch (err) {
       opts.onError?.(err as ConnectError);
     }
+    this.refreshToken(); // token expires once it's used to create a connected account. We need to get a new token for the next requests.
```

Comment:
This hack of refreshing tokens isn't the best but it's needed based on the existing implementation. Ideally, we could stop deleting the tokens.